engine/src/main/java/org/terasology/engine/subsystem/lwjgl/LwjglGraphics.java

```diff
@@ -254,6 +254,7 @@ private void checkOpenGL() {
                 GLContext.getCapabilities().OpenGL12,
                 GLContext.getCapabilities().OpenGL14,
                 GLContext.getCapabilities().OpenGL15,
+                GLContext.getCapabilities().OpenGL20,
```

Comment:
Hmmmm... maybe, but some of the opengl 2.0 code is quite technical. I.e. one of the first uses I've seen is a function to simply disable a shader. I'm not sure why that is used instead of methods available through earlier opengl versions, but this is something non visual and yet necessary in one form or another.

But I'd certainly like to get to a point where at least the ingame rendering code can be completely replaced, i.e. to allow for a renderer that uses opengl 4.x or, why not, even plug in a raytracer, for the joy of Flo and Immortius! =D